.github/workflows/release.yml

```diff
@@ -42,24 +45,30 @@ jobs:
         with:
           version: ${{ steps.previous_tag.outputs.tag }}
 
-      - name: Set development manifest
-        if: steps.semvers.outputs.v_patch == steps.get_current_tag.outputs.tag
-        run: |
-            jq --argjson icons "{\"16\": \"icons/dev/16x16.png\",\"48\": \"icons/dev/48x48.png\",\"128\": \"icons/dev/128x128.png\"}" '.icons = $icons | .browser_action.default_icon = $icons | .name = "Liquality Wallet - Dev"' ./src/manifest.json > ./src/manifest.tmp
-            mv ./src/manifest.tmp ./src/manifest.json
-
-      - name: Set development agents
-        if: steps.semvers.outputs.v_patch == steps.get_current_tag.outputs.tag
+    # Set the Amplitude Analytics only for Major prod releases.
+      - name: Set development agents (major for Prod)
+        if: steps.semvers.outputs.v_major == steps.get_current_tag.outputs.tag
         env:
           VUE_APP_AMPLITUDE_API_KEY: ${{ secrets.VUE_APP_AMPLITUDE_API_KEY }}
         run: |
             echo VUE_APP_AMPLITUDE_API_KEY =${{ secrets.VUE_APP_AMPLITUDE_API_KEY }} >> .env
-            sed -i "s/\/swap-testnet\/agent/\/swap-testnet-dev\/agent/g; s/\/swap\/agent/\/swap-dev\/agent/g" src/build.config.js
       - run: |
```

Comment:
Since you are repeating this with the below job, you can just put this as a separate step to deduplicate

Reply:
oh not the sed 😄 I mean the build step `npm run ci, npm run lint, npm run build`